src/Saml2/OasisAttributeNameUris.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace SocialiteProviders\Saml2;
+
+class OasisAttributeNameUris
+{
+    public const COMMON_NAME = 'urn:oid:2.5.4.3';
+    public const DISPLAY_NAME = 'urn:oid:2.16.840.1.113730.3.1.241';
+    public const GIVEN_NAME = 'urn:oid:2.5.4.42';
+    public const MAIL = 'urn:oid:0.9.2342.19200300.100.1.3';
+    public const ORGANIZATIONAL_UNIT_NAME = 'urn:oid:2.5.4.11';
+    public const PHONE = 'urn:oid:2.5.4.20';
+    public const PREFERRED_FIRST_NAME = 'urn:oid:1.2.840.113994.200.47';
+    public const PREFERRED_MIDDLE_NAME = 'urn:oid:1.2.840.113994.200.48';
+    public const PREFERRED_SURNAME = 'urn:oid:1.2.840.113994.200.49';
+    public const REGISTERED_GIVEN_NAME = 'urn:oid:1.2.840.113994.200.32';
```

Comment:
`REGISTERED_GIVEN_NAME` is not used anywhere

Reply:
Yes, some of these are not used anywhere. They are standardized URIs and I included them so developers can configure their `attribute_map`s with ease.

Do you think they should not be included? Or should I add some info about these in the readme? (I could also provide a pointer to the other collection (`ClaimTypes`) which includes an other standard as a note on how to configure custom attributes.)